src/main/kotlin/com/learnspigot/bot/entity/UserProfile.kt

```diff
@@ -26,11 +27,15 @@ data class UserProfile(
         reputation.add(point)
         acknowledgeRep(point, leaderboardManager, guild)
     }
-
+    fun changeRepuationToPing() {
+        reputationPing = !reputationPing
+    }
     private fun acknowledgeRep(point: ReputationPoint, leaderboardManager: LeaderboardManager, guild: Guild) {
         leaderboardManager.updateLeaderboardMessages()
         val channel = guild.getTextChannelById(System.getenv("SUPPORT_CHANNEL_ID"))!!
         if(point.postId != null) {
+            if (reputationPing)
+                channel.sendMessage("<@${id}>, you have got a reputation!").queue{ it.delete().queue{} }
```

Comment:
In reality, as this is something that gets deleted instantly and only serves as a way to notify the rep-receiver, it doesn't have to have a message outside of it just being `<@${id}>` so I would actually just remove the rest so it's only the mention in the message.